test/edu/ucsb/nceas/metacat/dataone/MNodeServiceIT.java

```diff
@@ -103,4363 +112,4494 @@
-
-            String st2 =
-                "<eml:eml xmlns:eml=\"eml://ecoinformatics.org/eml-2.1.1\" xmlns:xsi=\"http://www"
-                    + ".w3.org/2001/XMLSchema-instance\" packageId=\"tao.13397.1\" system=\"knb\" "
-                    + "xsi:schemaLocation=\"eml://ecoinformatics.org/eml-2.1.1 eml.xsd\">"
-                    + "<access" + " authSystem=\"knb\" order=\"allowFirst\">"
-                    + "<allow><principal>public"
-                    + "</principal><permission>read</permission></allow></access>" + "<dataset"
-                    + "><title>test2</title><creator "
-                    + "id=\"1445543475577\"><individualName><surName>test</surName"
-                    + "></individualName></creator>" + "<contact id=\"1445543479900"
-                    + "\"><individualName><surName>test</surName></individualName></contact"
-                    + "></dataset></eml:eml>";
-            Identifier newPid = new Identifier();
-            newPid.setValue(
-                "testUpdate." + (System.currentTimeMillis() + 1)); // ensure different from original
-            System.out.println("=================the new pid is " + newPid.getValue());
-            object = new ByteArrayInputStream(st2.getBytes(StandardCharsets.UTF_8));
-            SystemMetadata sysmeta2 = D1NodeServiceTest.createSystemMetadata(newPid, session.getSubject(), object);
-            sysmeta2.setFormatId(
-                ObjectFormatCache.getInstance().getFormat("eml://ecoinformatics.org/eml-2.1.1")
-                    .getFormatId());
-            sysmeta2.setObsoletes(guid);
-            Checksum sum1 = sysmeta2.getChecksum();
-            sum1.setValue("foo checksum");
-            sysmeta2.setChecksum(sum1);
-            object = new ByteArrayInputStream(st2.getBytes(StandardCharsets.UTF_8));
-            // do the update and it should fail
+        /**
+         * Test object updating
+         */
+        @Test
+        public void testMissMatchedChecksumUpdateSciMetadata() {
+            D1NodeServiceTest.printTestHeader("testMissMatchedChecksumUpdateSciMetadata");
+
             try {
-                MNodeService.getInstance(request).update(session, guid, object, newPid, sysmeta2);
-                fail("we shouldn't get here since the checksum is wrong");
-            } catch (InvalidSystemMetadata ee) {
+                String st1 =
+                    "<eml:eml xmlns:eml=\"eml://ecoinformatics.org/eml-2.1.1\" xmlns:xsi=\"http://www"
+                        + ".w3.org/2001/XMLSchema-instance\" packageId=\"tao.13397.1\" system=\"knb\" "
+                        + "xsi:schemaLocation=\"eml://ecoinformatics.org/eml-2.1.1 eml.xsd\">"
+                        + "<access" + " authSystem=\"knb\" order=\"allowFirst\">"
+                        + "<allow><principal>public"
+                        + "</principal><permission>read</permission></allow></access>" + "<dataset"
```

Comment:
unnecessary concat: `</access>" + "<dataset`

Also consider using java 17 block quotes for this and also line 780, 836, & 857, for better readability?